### PR TITLE
* HomeScreen UI

### PR DESCRIPTION
**Pull request by themohabgamal**
## What’s the perfect GIF to sum up this PR?
ChefsKissFrenchChefGIF

## 📋 What’s Cooking in This PR?
 🆕 New Feature: HomeScreen UI 

 ## 📸 Visual Treat:
Screenshot_1727260896

## 👀 Review Checklist:
+ Added Demo images for test (will be removed in Logic stage)
+ initialized screen util package
+ Screen switcher screen which is used in navigation of the bottom nav bar
+ App colors added pink and grey
+ Text styles class that has the main app font and styles

## 🥳 Let’s make this PR shine! 🌟

### CODE REVIEW

File: lib/screens_switcher.dart [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:mentorship/core/theming/colors.dart';
+import 'package:mentorship/features/home/logic/home_cubit.dart';
+import 'package:mentorship/features/home/ui/screens/home_screen.dart';
+import 'package:persistent_bottom_nav_bar/persistent_bottom_nav_bar.dart';
+
+class ScreensSwitcher extends StatelessWidget {
```
Comment (omarahmedx14) [inline lib/screens_switcher.dart:9]: ScreensSwitcher isn't the proper name I guess.

File: lib/screens_switcher.dart [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          title: "Wishlist",
+          activeColorPrimary: AppColors.pinkColor.withOpacity(0.1),
+          inactiveColorPrimary: Colors.black,
+          activeColorSecondary: AppColors.pinkColor),
+    ];
+  }
+}
+
+// Dummy screens for demonstration
+class ScreenFour extends StatelessWidget {
```
Comment (omarahmedx14) [inline lib/screens_switcher.dart:107]: consider making these dummy classes in other files later.

**Review by omarahmedx14 — CHANGES_REQUESTED** [review]
LGTM!

Thank you for this great PR, left small comments.

File: lib/screens_switcher.dart (hunk 1 above)
Comment (themohabgamal) [reply lib/screens_switcher.dart:9]: I thought about it's functionality (to have the bottom nav bar and to switch between screens) But ig MainScreen is better?

**Review by omarahmedx14 — APPROVED** [review]
LGTM!

**Pull request merged**